fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/virtualcolumns/ExpressionVirtualColumn.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.virtualcolumns;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * VirtualColumn to provide queryable column views created from a set of columns during a query.
+ */
+public class ExpressionVirtualColumn implements VirtualColumn {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ExpressionVirtualColumn.class);
+
+    private final String name;
+    private final String expression;
+    private final String outputType;
```

Comment:
https://druid.apache.org/docs/latest/querying/virtual-columns.html

We might want to defensively make an inner enum for the types on the virtual column.